packages/npm-history/src/index.test.ts

```diff
@@ -1,90 +1,3 @@
-/* EXPERIMENTAL */
```

Comment:
this file was renamed to `index.experimental.ts` here https://github.com/artalar/reatom/pull/1026